torchvision/transforms/v2/functional/_geometry.py

```diff
@@ -150,6 +163,15 @@ def _vertical_flip_image_pil(image: PIL.Image.Image) -> PIL.Image.Image:
     return _FP.vflip(image)
 
 
+def _vertical_flip_image_cvcuda(image: "cvcuda.Tensor") -> "cvcuda.Tensor":
+    cvcuda = _import_cvcuda()
```

Comment:
We shouldn't have to run the import everytime a cvcuda kernel is called, if the kernel gets registered it will be available and imported already
(same comment from `_vertical_flip_image_cvcuda`)